src/internal/events/events.go

```diff
@@ -117,6 +118,7 @@ func (b Bus) Event(ctx context.Context, key string, data map[string]any) {
 	props := analytics.
 		NewProperties().
 		Set(repoID, b.repoID).
+		Set(tenantID, b.tenant).
```

Comment:
`tenantID` is also needed as a Trait in the Identify call

Reply:
Okay, will re-add.